### PR TITLE
Display user avatar in leaderboard 

### PR DESCRIPTION
**Pull request by MrlolDev**
Update to #1023
Using dicebear to generate initial avatar or saved avatar_url

### CODE REVIEW

File: website/next.config.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ const nextConfig = {
         protocol: "https",
         hostname: "**.discordapp.com",
       },
+      {
+        protocol: "https",
+        hostname: "api.dicebear.com",
```
Comment (AbdBarho) [inline website/next.config.js:15]: this is not necessary, you can leave the image blank if the user has no image.

**Review by AbdBarho — COMMENTED** [review]
Thanks

File: website/src/components/LeaderboardTable/LeaderboardTable.tsx [hunk 2]
```diff
@@ -61,14 +62,34 @@ export const LeaderboardTable = ({
...
-          isAdminOrMod ? (
-            <Link as={NextLink} href={`/admin/manage_user/${row.original.user_id}`}>
-              {getValue()}
-            </Link>
-          ) : (
-            getValue()
-          ),
+        cell: ({ getValue, row }) => (
+          <div className="flex flex-row items-center gap-2">
+            {row.original.image ? (
```
Comment (AbdBarho) [inline website/src/components/LeaderboardTable/LeaderboardTable.tsx:67]: is it possible to only ternary on the url?

File: website/src/components/LeaderboardTable/LeaderboardTable.tsx [hunk 3]
```diff
@@ -61,14 +62,34 @@ export const LeaderboardTable = ({
...
+              <Image
+                src={`${row.original.image}`}
+                alt={`${getValue()}'s avatar`}
+                width={30}
+                height={30}
+                className="rounded-full"
+              />
+            ) : (
+              <Image
+                src={`https://api.dicebear.com/5.x/initials/png?seed=${getValue()}&radius=50&backgroundType=gradientLinear`}
```
Comment (rjmacarthy) [inline website/src/components/LeaderboardTable/LeaderboardTable.tsx:77]: I noticed that you are calling `getValue` more than once, maybe should we cache it?  I realise this prevents the explicit return.  Additionally, might it make sense to create a component for this i.e `<UserAvatar displayName={displayname} />` incase we want to use it again?

**Review by notmd — APPROVED** [review]
thanks!

**Pull request merged**